webpack.config.js

```diff
@@ -14,7 +14,7 @@ const plugins = []
 if (PRODUCTION) {
   plugins.push(
     new webpack.DefinePlugin({ 'process.env.NODE_ENV': JSON.stringify(ENVIRONMENT) }),
-    new webpack.optimize.ModuleConcatenationPlugin(),
```

Comment:
Please put the trailing `,` back in. :) It's required by the linting rules set up for this lib.

Reply:
o sure :)